From 66795ee7813dfc1e24d4fb4e87810b9f31a62898 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 21 Aug 2015 17:10:23 +0800
Subject: [PATCH 0275/5242] MLK-11395-8 video: mipi_dsi_samsung: add build
 support

commit  14d21767231749eeb3f1507da7788b7d3ce501e2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add build support for this new samsung's mipi dsi.
The old mipi dsi build option is removed from the
kernel default config. Instead, add the new mipi
dsi build option to the default config.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/Kconfig  |   10 ++++++++++
 drivers/video/fbdev/mxc/Makefile |    2 ++
 2 files changed, 12 insertions(+)

diff --git a/drivers/video/fbdev/mxc/Kconfig b/drivers/video/fbdev/mxc/Kconfig
index e9feb18..cfdb975 100644
--- a/drivers/video/fbdev/mxc/Kconfig
+++ b/drivers/video/fbdev/mxc/Kconfig
@@ -17,3 +17,13 @@ config FB_MXC
 config FB_MXC_SYNC_PANEL
         depends on FB_MXC
         tristate "Synchronous Panel Framebuffer"
+
+config FB_MXC_MIPI_DSI_SAMSUNG
+        tristate "MXC MIPI_DSI_SAMSUNG"
+        depends on FB_MXC_SYNC_PANEL
+        depends on FB_MXS
+
+config FB_MXC_TRULY_WVGA_SYNC_PANEL
+        tristate "TRULY WVGA Panel"
+        depends on FB_MXC_SYNC_PANEL
+        depends on FB_MXC_MIPI_DSI_SAMSUNG
diff --git a/drivers/video/fbdev/mxc/Makefile b/drivers/video/fbdev/mxc/Makefile
index 568574e..5fcecc7 100644
--- a/drivers/video/fbdev/mxc/Makefile
+++ b/drivers/video/fbdev/mxc/Makefile
@@ -1 +1,3 @@
 obj-$(CONFIG_FB_MXC_SYNC_PANEL) += mxc_dispdrv.o
+obj-$(CONFIG_FB_MXC_MIPI_DSI_SAMSUNG)           += mipi_dsi_samsung.o
+obj-$(CONFIG_FB_MXC_TRULY_WVGA_SYNC_PANEL)      += mxcfb_hx8369_wvga.o
-- 
1.7.9.5

